### ISSUE TITLE
Request: ELI5 level tutorial for using Clay as a Unet backbone

### ISSUE THREAD

**Issue by pkopparla** [Clay-foundation/model#302]
Hello,

I work a lot with Unets for satellite imagery for segmentation tasks, which is a popular satellite data/AI task. I'm interested in swapping out the typical Resnet/VGG encoder backbones with Clay to see how that changes performance. Would it be possible to make a tutorial notebook showing how to plug in the Clay encoder for Sentinel-2 RGB? If such an example exists in code somewhere, I'm happy to be pointed in that direction.

**Comment by srmsoumya** [comment]
The [Clay documentation page](https://clay-foundation.github.io/model/) provides more details on using the Clay Encoder for tasks like segmentation, detection, regression, and classification.

We also held a [technical walkthrough session](https://www.youtube.com/live/Zd3rbBj56P0) that covers the architecture and design language of Clay. Additionally, we have a basic [example of segmenting Clay on the Chesapeake Bay dataset here](https://github.com/Clay-foundation/model/tree/main/finetune/segment).
